### ISSUE TITLE
Add new line before @include 

### ISSUE THREAD

**Issue by TheElegantCoding** [stylelint-stylistic/stylelint-stylistic#46]
### What is the problem you're trying to solve?

This is considered a problem

```scss
.section-hero
{
  padding-bottom: $spacing_64;
  @include responsive(small_desktop)
  {
    flex-direction: row;
    padding-top: $spacing_160;
  }
}
```

### What solution would you like to see?

This is the correct way

```scss
.section-hero
{
  padding-bottom: $spacing_64;

  @include responsive(small_desktop)
  {
    flex-direction: row;
    padding-top: $spacing_160;
  }
}
```

**Comment by MorevM** [comment]
Hello @TheElegantCoding 

As [written in the repository documentation](https://github.com/stylelint-stylistic/stylelint-stylistic/blob/main/docs/developer-guide/rules.md#define-the-rule), the rules should refer to standard CSS syntax. 
The change you are requesting applies _exclusively_ to SASS.
For such validation in standard CSS there is a core rule [`at-rule-empty-line-before`](https://stylelint.io/user-guide/rules/) (I don't know why they don't consider it a stylistic rule by the way).

It's not actually a big deal to add support for block SASS-mixins in there, but I believe there should be a single source of truth for similar rules.

I would recommend opening [an issue in Stylelint](https://github.com/stylelint/stylelint/issues) with suggestions:
1) Either add support for block SASS-mixins to `at-rule-empty-line-before` (I expect they won't do this for the reason described in the first paragraph). By the way, it already works with SASS mixins - it just doesn't distinguish between block/inline of them, so the enchancement will only deal with the differences between block and inline at-rules, probably by introducing a new option.
2) Either mark the rule as stylistic (which it is) - then it will be ported to this repository, in which case I wouldn't mind adding such support, but here. 

You could also try opening the same issue in the [`stylelint-scss`](https://github.com/stylelint-scss/) repository (since your request is only about SCSS), but in general I expect the answer to be about the same.

**Comment by TheElegantCoding** [comment]
you are right the answer is to use `at-rule-empty-line-before`, thanks
